gui/main.py

```diff
@@ -7,8 +7,10 @@
 import os.path
 import json
 import logging
+import sys
 
 # Minibot imports.
+sys.path.insert(0, basestation)
```

Comment:
Why do we have this here again?

Reply:
hm i think that line is unnecessary; I added it when basestation wouldn't run on my computer due to some path issue